<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomatoClock</class>
 <widget class="QMainWindow" name="TomatoClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TomatoClock</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>951</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Todolist">
     <attribute name="title">
      <string>Todolist</string>
     </attribute>
     <widget class="QPushButton" name="Add">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>290</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Manage">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>380</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="tasksdetail">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>591</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="addlabel">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>40</y>
        <width>601</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Completed">
     <attribute name="title">
      <string>Completed</string>
     </attribute>
     <widget class="QPushButton" name="completed_Delete">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>280</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="completed_Clear">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>380</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="completedlabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>401</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Clock">
     <attribute name="title">
      <string>Clock</string>
     </attribute>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Stop">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Finish">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>317</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLCDNumber" name="minute_clock">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="second_clock">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="clock_tasksname">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>current task:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clock_tasksid">
      <property name="geometry">
       <rect>
        <x>302</x>
        <y>250</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>work time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>75</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>relax time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="work_time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="relax_time">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Done">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>360</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
